recipe/meta.yaml

```diff
@@ -110,3 +107,4 @@ extra:
     - msarahan
     - ocefpaf
     - mingwandroid
+    - flaviomartins
```

Comment:
👍🏽 

(Our policy is, you broke it, you bought it ;-p. Jokes aside, thanks for fixing this package!)